composer.json

```
@@ -9,11 +9,15 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require-dev": {
-        "simplemachines/build-tools": "dev-release-3.0"
+        "simplemachines/build-tools": "dev-release-3.0",
+        "friendsofphp/php-cs-fixer": "^3.40"
+    },
+    "scripts": {
+        "lint": "php-cs-fixer fix -v"
```

```suggestion
        "lint": "php-cs-fixer check -v"
```

I don't think these workflows are the place to make changes to files. Unless I misunderstand something fundamental about how these workflows function, all we can do here is check the files.

---

That is, unless these workflows can automatically add an extra commit with the changes. If that's the case, that would be awesome.

---

I think this is just so you can run 'composer lint' in your command line and it will run the php-cs-fixer.  But I would suggest it run it only against your changed files.  Existing files not compliant should be ignored.